run_blockchain_terraformer.py

```diff
@@ -683,20 +683,13 @@ def organize_pool_details_solidly_v2(
 
     if "velocimeter" in exchange:
         default_fee = factory_contract.caller.getFee(pool_address)
-        default_fee = float(default_fee)
-        if default_fee <= 1000:
-            default_fee = default_fee / 10000
-        else:
-            default_fee = default_fee / 10 ** 18
-        default_fee = default_fee / 10 ** 18
+        default_fee = float(default_fee) / 10000
     elif "scale" in exchange:
         default_fee = factory_contract.caller.getRealFee(pool_address)
-        default_fee = float(default_fee)
-        default_fee = default_fee / 10 ** 18
+        default_fee = float(default_fee) / 10 ** 18
     else:
         default_fee = factory_contract.caller.getFee(pool_address, is_stable)
-        default_fee = float(default_fee)
-        default_fee = default_fee / 10000
+        default_fee = float(default_fee) / 10000
```

Comment:
Why do we have the same code replicated in two different places?

Reply:
Between solidly_v2 and the terraformer? If that's the question - it's because the terraformer was intended to be run independently. 

If the question is why do we call getFee twice - it's because the functions have different inputs.

Reply:
No, my point was that we should use the same function in both places rather than replicating it

Reply:
I see your point, but it would require a major refactor of the terraformer file.